Cabal/doc/installing-packages.rst

```diff
@@ -151,7 +151,14 @@ repository.
     repository my-local-repository
       url: file+noindex:///absolute/path/to/directory
 
-``cabal`` will construct the index automatically from the
+Given that ``/absolute/path/to/directory`` looks like
+::
+
+    /absolute/path/to/directory/
+        foo-0.1.0.0.tar.gz
+        bar-0.2.0.0.tar.gz
```

Comment:
I would
- also add `bar-0.2.0.0.cabal` file 
- reorder this to first say what was already i.e. *`cabal`` will ...`, and then
- add:

For example from the directory contents

```
foo-0.1.0.0.tar.gz
bar-0.2.0.0.tar.gz
bar-0.2.0.0.cabal
```

`cabal` will create an index with two packages, `foo-0.1.0.0` and `bar-0.2.0.0` using in-tarball `.cabal` file for `foo` and `bar-0.2.0.0.cabal` filefor foo.

TL;DR first tell what, then show an example. We should avoid examples as part of "specification".